src/demo/DemoTestLoCService.java

```diff
@@ -306,4 +308,46 @@ private static void testdeleteAndAddSavingsAccounts(String lineOfCreditId, List<
 
 		}
 	}
+	
+	// tests PATCHing a line of credit
+	public static void testPatchLineOfCredit() throws MambuApiException{
+		String methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println("\nIn " + methodName);
+		
+		LinesOfCreditService linesOfCreditService = MambuAPIFactory.getLineOfCreditService();
+
+		final String clientId = DemoUtil.getDemoClient().getId();
+		// Get Lines of Credit for a client
+		List<LineOfCredit> clientLoCs = linesOfCreditService.getClientLinesOfCredit(clientId, 0, 5);
+		System.out.println(clientLoCs.size() + " lines of credit  for Client " + clientId);
+		
+		if(clientLoCs.isEmpty() && clientLoCs.get(0) == null){
```

Comment:
We should use the OR operator here instead of AND. If the first part of the condition is true, meaning that the list of lines of credit is empty, then there is no reason to check whether the first item ( .get(0)) of the collection is null or not. On the other hand, if the collection is not empty, the first element might be null. Therefore, I suggest to transform the condition into something like `clientLoCs.isEmpty() || clientLoCs.get(0) == null`.